src/components/Grid/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Juan Carlos Medina
```

Comment:
Even though I practically rewrote this, I felt it was correct thing to leave his original license. See my thoughts here: https://github.com/getethos/ethos-design-system/pull/194/files#diff-515fd4503108cc3de9a70f41c8aba2dcR2